test/alphabet/nucleotide/CMakeLists.txt

```diff
@@ -1,15 +1,15 @@
 add_executable(dna5_test dna5_test.cpp)
```

Comment:
I repeat:
  * the changes you are proposing are either contained in another open PR or are obsoleted by another open PR.
  * furthermore I don't see that they are *necessary* for this PR, in the sense that without them tests would break (in that case you would of course need to include them)
  * therefore the least invasive change requiring the least amount of conflict resolution is just not including the changes

If you disagree, please argue along these lines and not with attitude.